tssc/step_implementers/container_image_static_compliance_scan/openscap.py

```diff
@@ -144,41 +143,51 @@ def _run_step(self): # pylint: disable=too-many-locals
                 _err=sys.stderr,
                 _tee=True
             )
-            container_id = result.stdout.rstrip()
+            container_id = buildah_containers_result.stdout.rstrip()
             print(f"container_id to scan = {container_id}")
 
+            # baking `buildah unshare` command to wrap other buildah commands with
```

Comment:
Nifty!

Reply:
indeed. made testing a bit of a pain. but meh.

Reply:
hopefully Python string don't break this though.